manifests/init.pp

```diff
@@ -70,8 +69,8 @@
     owner   => 'root',
     group   => $cert_dir_group,
     mode    => $cert_dir_mode,
-    purge   => $purge_unmanaged_CAs,
-    recurse => $purge_unmanaged_CAs,
+    purge   => $purge_unmanaged_cas,
+    recurse => $purge_unmanaged_cas,
```

Comment:
I've been out of the game for a minute. Are puppet parameters case sensitive? I'm only asking so I can make sure that a release with this change follows semver correctly (i.e. is this a breaking change?)

Reply:
Variables are case sensitive, yes. https://www.puppet.com/docs/puppet/8/lang_variables#lang_var_naming-variable-names

Reply:
Only did it because puppet-lint does complain otherwise:
```
pdk (WARNING): puppet-lint: variable contains a capital letter (manifests/init.pp:46:11)
pdk (WARNING): puppet-lint: variable contains a capital letter (manifests/init.pp:73:16)
pdk (WARNING): puppet-lint: variable contains a capital letter (manifests/init.pp:74:16)
```

So the argument is simply: because of style guides ;)
Sure, the alternative could be to deactivate the checks instead.

Reply:
I kind of lean towards staying backwards compatible, but don't feel so strongly that I can't be convinced otherwise. If we were breaking other stuff that wasn't as simple as a disabled check I might feel differently I guess. Whatever you all think we should do works for me.

Reply:
Changed the commit so that checks for "puppet-lint: variable contains a capital letter" are ignored.
Maybe this could be changed in the next major release :)